spec/features/idv/phone_spec.rb

```diff
@@ -68,6 +68,39 @@
     end
   end
 
+  context 'failing to verifiy 2FA phone' do
+    scenario 'requires verifying and confirming a different phone', :idv_job do
+      user = create(
+        :user, :signed_up,
+        phone: '+1 (555) 555-5555',
```

Comment:
Should we add a comment here that this phone is expected to fail verification?

Reply:
Or maybe instead of `fill_in 'Phone', with: user.phone` below on line 86, create a new method called `enter_phone_number_that_will_fail_verification`?